locations/spiders/choicehotels.py

```diff
@@ -1,79 +1,75 @@
 import json
 import re
-import scrapy
-
 from locations.items import GeojsonPointItem
+from scrapy.spiders import SitemapSpider
 
 
-brand_name_override = {
-    "Comfort Suites Suites": "Comfort Suites",
-    "Econo Lodge Lodge": "Econo Lodge",
-}
-
-
-class ChoiceHotelsSpider(scrapy.Spider):
+class ChoiceHotelsSpider(SitemapSpider):
     name = "choicehotels"
     item_attributes = {"brand": "Choice Hotels", "brand_wikidata": "Q1075788"}
     allowed_domains = ["choicehotels.com"]
     download_delay = 0.2
-
-    base_url = "https://www.choicehotels.com/cms/pages/choice-hotels"
-
-    start_urls = [
-        "https://www.choicehotels.com/cms/pages/comfort-inn/sitemap?d=DESKTOP&applocale=en-us",
-        "https://www.choicehotels.com/cms/pages/comfort-suites/sitemap?d=DESKTOP&applocale=en-u",
-        "https://www.choicehotels.com/cms/pages/quality-inn/sitemap?d=DESKTOP&applocale=en-u",
-        "https://www.choicehotels.com/cms/pages/sleep-inn/sitemap?d=DESKTOP&applocale=en-u",
-        "https://www.choicehotels.com/cms/pages/clarion/sitemap?d=DESKTOP&applocale=en-u",
-        "https://www.choicehotels.com/cms/pages/cambria/sitemap?d=DESKTOP&applocale=en-u",
-        "https://www.choicehotels.com/cms/pages/mainstay/sitemap?d=DESKTOP&applocale=en-u",
-        "https://www.choicehotels.com/cms/pages/suburban/sitemap?d=DESKTOP&applocale=en-u",
-        "https://www.choicehotels.com/cms/pages/econo-lodge/sitemap?d=DESKTOP&applocale=en-u",
-        "https://www.choicehotels.com/cms/pages/rodeway-inn/sitemap?d=DESKTOP&applocale=en-u",
+    user_agent = "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/103.0.0.0 Safari/537.36"
```

Comment:
Is this a spider attribute? I thought it needed to be custom_settings

Reply:
Didn't know that. I also think we should unify on some browser user agents, so we can update them more regularly and not have ancient ones sitting around. But I don't know how important it is, and it's not very high on my todo list.